web/styles/left_sidebar.css

```diff
@@ -761,6 +761,10 @@ li.topic-list-item {
     visibility: hidden;
 }
 
+.zero-topic-unreads .unread_mention_info {
+    margin-right: 20px;
```

Comment:
How is this 20px computed? It'd be ideal if we can use a CSS variable to have this be defined in a way that it will always match the margin we'd have if there were unread mentions in the topic.